extension.json

```diff
@@ -33,4 +33,4 @@
 		"ParserFirstCallInit": "CurseTwitterHooks::onParserFirstCallInit"
 	},
 	"manifest_version": 2
-}
\ No newline at end of file
+}
```

Comment:
Adding a new line here doesn't break MediaWiki's extension loader?

Reply:
No i doesn't, and adding the new line will give the file a proper EOL. It does not break the Josn format and it makes the file compliant to *nix formatting standards..